clients/java/src/main/java/io/camunda/zeebe/client/ClientProperties.java

```diff
@@ -119,7 +119,8 @@ public final class ClientProperties {
   public static final String CLOUD_CLIENT_SECRET = "zeebe.client.cloud.secret";
 
   /**
-   * @see ZeebeClientCloudBuilderStep1.ZeebeClientCloudBuilderStep4#withRegion(java.lang.String)
+   * @see
+   *     io.camunda.zeebe.client.ZeebeClientCloudBuilderStep1.ZeebeClientCloudBuilderStep2.ZeebeClientCloudBuilderStep3.ZeebeClientCloudBuilderStep4#withRegion(String)
```

Comment:
🔧 This looks like an unnecessary qualifier

Reply:
It was auto generated :shrug: Without it IntelliJ kept marking it as an error :thinking: